framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -164,6 +165,13 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 		this._chainConnectorStore.close();
 	}
 
+	public _getCcuFee(tx: Record<string, unknown>): bigint {
+		if (this.config.ccuFee && this.config.ccuFee !== '0') {
+			return BigInt(this.config.ccuFee);
+		}
+		return transactions.computeMinFee(tx);
```

Comment:
this function accepts params schema , otherwise it will skip params into the computation